pom.xml

```diff
@@ -13,7 +13,7 @@
     </organization>
     <licenses>
         <license>
-            <name>MIT License</name>
+            <name>Apache License 2.0</name>
             <url>http://www.opensource.org/licenses/mit-license.php</url>
```

Comment:
@luxu1-ms / @shivsood please update the URL as well:
http://www.apache.org/licenses/LICENSE-2.0.txt

Other than that perfect thanks!

Reply:
Thank you @rajmera3 Done